src/angular/header-lean/app-chooser-section.scss

```diff
@@ -47,4 +47,25 @@
   &:last-child {
     border-bottom: var(--sbb-border-width-thin) solid var(--sbb-color-cloud);
   }
+
+  .sbb-icon,
+  img {
```

Comment:
```suggestion
  :is(.sbb-icon, img) {
```

Reply:
should be :is here, because it's nested in `a` and it would generate duplicated definitions on sass compilation.